dev/merge_spark_pr.py

```diff
@@ -31,7 +31,15 @@
 import subprocess
 import sys
 import traceback
-import urllib2
+try:
+    from urllib2 import urlopen
+    from urllib2 import Request
+    from urllib2 import HTTPError
+except ImportError:
+    # Python 3
+    from urllib.request import urlopen
+    from urllib.request import Request
+    from urllib.error import HTTPError
```

Comment:
Instead of `try`, shall we move this to `if` statement at line 50 since we already have a branch for versioning?
```python
if sys.version < '3':
    input = raw_input  # noqa
```